src/views/Viewer.vue

```diff
@@ -977,6 +1014,47 @@ export default {
 		handleTrapElementsChange(element) {
 			this.trapElements.push(element)
 		},
+
+		// Support full screen API on standard-compliant browsers and Safari (apparently except iPhone).
+		// Implementation based on:
+		//   https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API/Guide
+
+		requestFullscreen() {
+			const el = document.documentElement
+			if (el.requestFullscreen) {
+				el.requestFullscreen()
+			} else if (el.webkitRequestFullscreen) {
+				el.webkitRequestFullscreen()
+			}
+		},
+
+		exitFullscreen() {
+			if (document.exitFullscreen) {
+				document.exitFullscreen()
+			} else if (document.webkitExitFullscreen) {
+				document.webkitExitFullscreen()
+			}
```

Comment:
```suggestion
			document?.exitFullscreen?.() || document?.webkitExitFullscreen?.()```

Reply:
The `exitFullscreen()` method returns a `Promise` - I am not sure if the suggested change preserves the intended behaviour.

Reply:
Good point, you can leave as is then :)